CreeDictionary/search_quality/sample.csv

```diff
@@ -1,70 +1,384 @@
+beaver berries,amiskomin,amiskominak,,,
+berries,mînisa,mînis,,,
+berry,mînis,mînisa,,,
+berri,mînis,mînisa,,,
+stove pipe,okohtaskwahikan,,,,
+stove-pipe,okohtaskwahikan,,,,
+stovepipe,okohtaskwahikan,,,,
+in the area,cîkâhtaw,,,,
+bread,pahkwêsikan,,,,
+yellow hat,osâwastotin,,,,
+sleep,nipâw,,,,
+nîminâniwan,nîminâniwan,nîmiw,,,
+I love you,kisâkihitin,,,,
+usual,mâna,,,,
+no,namoya,,,,
+this,ôma,awa,ôhi,,
+that,anima,,,,
+good morning,miyo-kîkisêpâyâw,,,,
+me,niya,,,,
+good bye,ekosi mâka ,,,,
+goodbye,ekosi mâka ,,,,
+work together,mâmawi-isîhcikêwin,,,,
+working together,mâmawi-isîhcikêwin,,,,
+work togeth,mâmawi-isîhcikêwin,,,,
+Cree,nêhiyaw,nêhiyawêwin,,,
+Edmonton,amiskwaciy-wâskahikan,,,,
+edmonton,amiskwaciy-wâskahikan,,,,
+hill,waciy,ispatinaw,,,
+beaver hill,natimîwiyiniwak,amiskwacîwiyiniwak,,,
+beaver hills,natimîwiyiniwak,amiskwacîwiyiniwak,,,
+Calgary,otôskwanihk,,,,
+calgary,otôskwanihk,,,,
+car,sêhkêpayîs,,,,
+water,nipiy,,,,
+telling a story,âcimêw,âcimow,,,
+on water,nipîhk,,,,"aspirational search, as result is based on inflections, not a direct definition entry"
+hello,tânisi,,,,
+difficult thing,âyiman,,,,
+many snakes,kinêpikoskâw,,,,
+there are many snakes,kinêpikoskâw,,,,
+dances,nîmiw,nîmihitow,,,
+dancing time,nîminâniwan,nîmiw,,,
+go to bed,mâci-nipa,,,,
+Merry Christmas,miyo-manitowi-kîsikânisik,,,,
+merry christmas,miyo-manitowi-kîsikânisik,,,,
+i see you,kiwâpamitin,,,,
+he sees him,wâpamêw,,,,
+she sees him,wâpamêw,,,,
+starblanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+star blanket,atâhkakohp,acâhkosa kâ-otakohpit,,,
+being taught,kiskinwahamâkosiw,,,,
+they see us,niwâpamikonânak,kiwâpamikonawak,,,
+gym,mêtawêwikamik,,,,This item and below are from NS152
+lake,sâkahikan,,,,
+juice,sîwâpoy,,,,
+hospital,âhkosîwikamik,,,,
+sock,asikan,,,,
+pail,askihk,,,,
+potato,askipwâwi,,,,
+dog,atim,nitêm,,,very irregular inflections
+doll,awâsisihkân,,,,
+tobacco,ciscêmâs,cistêmâw,,,
+fork,cîscahikanis,,,,
+girl,iskwêsis,,,,
+woman,iskwêw,,,,
+fish,kinosêw,,,,
+bacon,kohkôsiwiyin,,,,
+corn,mahtâmin,,,,
+pencil,masinahikanâhtik,,,,
+slate,masinahikanâpisk,,,,
+pen,masinahikanâpiskos,,,,
+tea,maskihkîwâpoy,,,,
+bear,maskwa,,,,
+glass,minihkwâcikan,minihkwêwiyâkanis,,,
+cup,minihkwâcikanis,,,,
+cat,minôs,,,,
+pants,mitâs,,,,
+feather,mêkwan,,,,
+road,mêskanaw,,,,
+soup,mîcimâpoy,,,,
+bead,mîkis,,,,
+knife,môhkomân,,,,
+little knife,môhkomânis,,,,
+moose,môswa,,,,
+toboggan,napakâhtik,,,,
+frying pan,napwên,,,,
+goose,niska,,,,
+boy,nâpêsis,,,,
+man,nâpêw,,,,
+Cree person,nêhiyaw,,,,
+tomato,okiniy,,,,
+little carrot,oskâcâskos,,,,
+carrot,oskâtâsk,,,,
+nut,pakân,,,,
+pepper,papêskomina,,,,
+apple,picikwâs,,,,
+ball,pâkâhtowân,,,,
+sun,pîsim,,,,
+skunk,sikâk,,,,
+sugar,sîwinikan,,,,
+salt,sîwîhtâkan,,,,
+money,sôniyâw,,,,
+butter,tohtôsâpoy,,,,
+milk,tohtôsâpoy,,,,
+fur,wayân,,,,
+dish,wiyâkan,,,,
+banana,wâkâs,,,,
+rabbit,wâpos,,,,
+egg,wâwi,,,,
+little spoon,êmihkwânis,,,,
+brings it,pêtawêw,,,,
```

Comment:
brings it: also pêtâw
See also brings below, would like to see these return similar Cree words